### PR TITLE
ICU-21546 BRS69GA Update version numbers for 69GA

### PR DESCRIPTION
**Pull request by erik0686**
##### Checklist

- [x] Required: Issue filed: https://unicode-org.atlassian.net/browse/ICU-21546
- [x] Required: The PR title must be prefixed with a JIRA Issue number. 
- [x] Required: The PR description must include the link to the Jira Issue, for example by completing the URL in the first checklist item
- [x] Required: Each commit message must be prefixed with a JIRA Issue number. 
- [x] Issue accepted (done by Technical Committee after discussion)
- [ ] Tests included, if applicable
- [ ] API docs and/or User Guide docs changed or added, if applicable

### CODE REVIEW

File: icu4j/build.properties [hunk 1]
```diff
@@ -6,7 +6,7 @@
 #*******************************************************************************
 api.report.version = 69
 api.report.prev.version = 68
-release.file.ver = 69rc
-api.doc.version = 69.1 Release Candidate
-maven.pom.ver = 69.1-SNAPSHOT
+release.file.ver = 69.1
```
Comment (daniel-ju) [inline icu4j/build.properties:9]: This should be `69_1` instead of `69.1`.

**Review by daniel-ju — APPROVED** [review]

**Review by jefgen — APPROVED** [review]

**Comment by erik0686** [comment]
@daniel-ju @jefgen Do you happen to know if the appveyor failure is from another PR, or if it could be cause by my changes?

**Comment by jefgen** [comment]
> Do you happen to know if the appveyor failure is from another PR, or if it could be cause by my changes?

Sigh... AppVeyor has been extremely flaky recently, so I suspect that it likely had nothing to do with your change.

FWIW, the failure is a random crash:

```
make[1]: *** [../config/mh-cygwin64:76: uresdata.o] Segmentation fault
make[1]: Leaving directory '/cygdrive/c/projects/icu/icu4c/source/common'
make: *** [Makefile:153: all-recursive] Error 2
Command exited with code 2
```

I'll trigger a re-build of this PR with AppVeyor and we can see if it passes on rebuild.

**Comment by jefgen** [comment]
FYI: It looks like the AppVeyor build passed on re-run, so it was just flakiness.

**Pull request merged**